sdks/java/core/src/main/java/org/apache/beam/sdk/util/BigQueryTableInserter.java

```diff
@@ -95,6 +94,22 @@ public BigQueryTableInserter(Bigquery client) {
     this.client = client;
```

Comment:
Yes. Beam is in active development and we are taking on backwards incompatible changes at the time.